### DISCUSSION TITLE
Group validation

### DISCUSSION THREAD

**Discussion by terafirmanz** [directus/directus#19668] [Feature Requests]
### Summary

When placing fields into a group it would be nice to be able to set validation on the group that can run rules based on all fields in the group.

### Basic Example

I place a number of boolean fields into a group that can have a combination but I want to state that at least 1 must be true but no more than 3 can be true.

### Motivation

Allow for validation of multiple fields in complex scenarios. Currently, the only way I can see is to use checkboxes but then your data is json or csv stored in the database and you can only keep boolean data. There are times when storing json might not be feasible for other apps using the data or more complex data than just a boolean might be required.

### Detailed Design

When wishing to have validation rules that span accross multiple fields you would place these fields into a group (any one of accordion / detail / raw) then on the groups properties under validation when adding rules you can change the applicable fields to be any one of the fields int eh group rather than only the group itself.

### Must Have:
- A the ability to implement validation rules that span multiple fields.

### Drawbacks

This would make the group object more than just a UI element. 

### Alternatives

A new field type of validation could be added that would allow for applying rules to the collection.
Alternatively a flow could be used but this makes for a more complicated and harder learning path for users to implement.

### Adoption Strategy

would not be a breaking change as the default would be no validation as it currently is but would allow users to adopt it if desired.

### Unresolved Questions

I have seen screenshots of validation rules that show multiple fields used but am unable to find how people have done this or if it is a different scenario as they all relate to users and roles.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
